ci/builders/README.md

```diff
@@ -316,6 +317,8 @@ In general any executable found in the path can be used as language. The
 default is empty which means no interpreter will be used to run the script
 and it is assumed the script is already an executable with the right
 permissions to run in the target platform.
+* **test_timeout_secs** - the timeout in seconds for the step running the test. This value overrides the
+default 1 hour timeout. It is recommended to add timeouts to tests leaking processes to allow LUCI services to collect logs.
```

Comment:
Maybe: "When debugging, or if a third-party program is known to misbehave, it is recommended...." to make it clear that the expectation is that tests should not be leaking processes.

Reply:
It's also not clear to me what the call to action here is, like should the 1 hour timeout be sufficient most of the time? In what cases would I want to change it?

Reply:
Updated!